c/include/libsbp/ndb.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
+ * Contact: Fergus Noble <fergus@swift-nav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+/*****************************************************************************
+ * Automatically generated from yaml/swiftnav/sbp/ndb.yaml
+ * with generate.py. Please do not hand edit!
+ *****************************************************************************/
+
+/** \defgroup ndb Ndb
+ *
+ *  * Messages for logging NDB events.
+ * \{ */
+
+#ifndef LIBSBP_NDB_MESSAGES_H
+#define LIBSBP_NDB_MESSAGES_H
+
+#include "common.h"
+#include "gnss.h"
+
+
+/** Navigation DataBase Event
+ *
+ * This message is sent out when an object is stored into NDB. If needed
+ * message could also be sent out when fetching an object from NDB.
+ */
+#define SBP_MSG_NDB_EVENT 0x0400
+typedef struct __attribute__((packed)) {
+  u64 recv_time;      /**< HW time in milliseconds. [ms] */
+  u8 event;          /**< Store = 1,
+Fetch = 2 
+ */
+  u8 object_type;    /**< Ephemeris = 1,
+Almanac = 2,
```

Comment:
We also store Almanac week number separately. Should we add it here and in other files?